src/EmbeddedManifold.jl

```diff
@@ -252,6 +252,20 @@ function decorator_transparent_dispatch(
 )
     return Val(:intransparent)
 end
+function decorator_transparent_dispatch(
+    ::typeof(distance),
+    ::AbstractEmbeddedManifold,
+    args...,
+)
+    return Val(:parent)
+end
+function decorator_transparent_dispatch(
+    ::typeof(distance),
+    ::AbstractEmbeddedManifold{𝔽,<:AbstractIsometricEmbeddingType},
```

Comment:
I think this should be restricted to `TransparentIsometricEmbedding`.
```suggestion
    ::AbstractEmbeddedManifold{𝔽,TransparentIsometricEmbedding},
```

Reply:
That's interesting. `exp` and `log` both have `TransparentIsometricEmbedding` here, while `norm` has `AbstractIsometricEmbeddingType`. All three depend on the metric though. Why the different overloads for these three?

Reply:
Actually, I have no idea why `norm` is `transparent` for `AbstractIsometricEmbedding`. As far as I understand it, `norm` should be `parent` for non-transparent embedding and `transparent` for transparent embedding. Maybe @kellertuer knows why it's different?

Reply:
Ah, right, I was wrong. It's correct. `AbstractIsometricEmbedding` means that `inner` and `norm` can be transparently forwarded while `exp` and `log` can only be directly forwarded for `TransparentIsometricEmbedding`. That's correct. That's basically what these embedding types mean.

Reply:
Still, `norm` should probably be `:parent` for `AbstractEmbeddedManifold`?

Reply:
@sethaxen Why did you change it back to `<:AbstractIsometricEmbeddingType`?

Reply:
Whoops, accident.